docs/framework/migration-guide/how-to-determine-which-versions-are-installed.md

```diff
@@ -163,7 +163,7 @@ Users can install and run multiple versions of the .NET Framework on their compu
 - The following example checks the value of the `Release` keyword to determine whether .NET Framework 4.6.2 or higher is installed, regardless of Windows OS version (returning `True` if it is and `False` otherwise).
 
     ```PowerShell
-    Get-ChildItem "HKLM:SOFTWARE\Microsoft\NET Framework Setup\NDP\v4\Full\" | Get-ItemPropertyValue -Name Release | ForEach-Object { $_ -ge 394802 } 
+    Get-ChildItem 'HKLM:\SOFTWARE\Microsoft\NET Framework Setup\NDP\v4\Full\' | Get-ItemPropertyValue -Name Release | Foreach-Object { $_ -ge 394802 } 
```

Comment:
@rpetrusha @randomnote1  Does the `Foreach-Object` have correct casing? Docs state `ForEach-Object`:
https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/foreach-object?view=powershell-6

Reply:
It does not. I missed that!